static/js/components/subject_page/main_pane.tsx

```diff
@@ -42,9 +41,7 @@ interface SubjectPageMainPanePropType {
 export function SubjectPageMainPane(
   props: SubjectPageMainPanePropType
 ): JSX.Element {
-  const placeType = props.place.types
```

Comment:
place_type is needed in the server side for trimming config, so moved the default value there. For different page, the default place type could be different? The disaster page uses "Earth" rather than "Place".

As a standalone component, it is better to use the value from the props?

Reply:
+1, we can move this to the server. add a TODO for now?

Reply:
this makes sense